rpcs3/Emu/RSX/VK/VKFragmentProgram.cpp

```diff
@@ -371,6 +371,7 @@ void VKFragmentProgram::Decompile(const RSXFragmentProgram& prog)
 
 void VKFragmentProgram::Compile()
 {
+	fs::create_path(fs::get_config_dir() + "/shaderlog");
 	fs::file(fs::get_config_dir() + "shaderlog/FragmentProgram.spirv", fs::rewrite).write(shader);
```

Comment:
I don't thinks so, but extending the fs::file class when the fs::rewrite is passed should create the path too, not just the file.
Than there is no need to call the create_path function everytime.
But thats my opinion.